server/datastore/mysql/labels.go

```diff
@@ -36,10 +39,54 @@ func (d *Datastore) ApplyLabelSpecs(specs []*kolide.LabelSpec) (err error) {
 			if s.Name == "" {
 				return errors.New("label name must not be empty")
 			}
-			_, err := stmt.Exec(s.Name, s.Description, s.Query, s.Platform, s.LabelType)
+			_, err := stmt.Exec(s.Name, s.Description, s.Query, s.Platform, s.LabelType, s.LabelMembershipType)
 			if err != nil {
 				return errors.Wrap(err, "exec ApplyLabelSpecs insert")
 			}
+
+			if s.LabelType == kolide.LabelTypeBuiltIn ||
+				s.LabelMembershipType != kolide.LabelMembershipTypeManual {
+				// No need to update membership
+				continue
+			}
+
+			var labelID uint
+			sql = `
+SELECT id from labels WHERE name = ?
+`
+			err = tx.Get(&labelID, sql, s.Name)
+			if err != nil {
```

Comment:
```suggestion
			if err := tx.Get(&labelID, sql, s.Name); err != nil {
```

I'm always ambivalent about it.